rotkehlchen/tests/unit/decoders/test_main.py

```diff
@@ -224,3 +225,216 @@ def test_simple_erc20_transfer(
+        type=0,
+        logs=[],
+    )
+    dbethtx = DBEthTx(database)
+    with database.user_write() as cursor:
+        dbethtx.add_ethereum_transactions(cursor, [transaction], relevant_address=None)
+        events = evm_transaction_decoder.decode_transaction(
+            write_cursor=cursor,
+            transaction=transaction,
+            tx_receipt=receipt,
+        )
+    assert events == [
+        HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=0,
+            timestamp=Timestamp(0),
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(),
+            location_label=from_address,
+            notes='Burned 0 ETH for gas',
+            counterparty='gas',
+            identifier=None,
+            extra_data=None,
+        ),
+        HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=1,
+            timestamp=0,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(0.5)),
+            location_label=from_address,
+            notes=f'Send 0.5 ETH to {to_address}',
+            counterparty=to_address,
+            identifier=None,
+            extra_data=None,
+        ),
+    ]
+
+
+@pytest.mark.parametrize('ethereum_accounts', [['0xa4A6A282A7fC7F939e01D62D884355d79f5046C1']])
+def test_eth_deposit(
+        database,
+        ethereum_accounts,
+        evm_transaction_decoder,
+):
+    """
+    Data taken from
+    https://etherscan.io/tx/0x8f91a9b98a856282cdad74d9b8a683504c13e3c9d810e4e22bd0ca2eb9d71800
+    """
+    evmhash = deserialize_evm_tx_hash('0x8f91a9b98a856282cdad74d9b8a683504c13e3c9d810e4e22bd0ca2eb9d71800')  # noqa: E501
+    from_address = ethereum_accounts[0]
+    to_address = '0xAe2D4617c862309A3d75A0fFB358c7a5009c673F'  # Kraken 10
+    transaction = EvmTransaction(
+        tx_hash=evmhash,
+        timestamp=0,
+        block_number=0,
+        from_address=from_address,
+        to_address=to_address,
+        value=100000000000000000000,
+        gas=0,
+        gas_price=0,
+        gas_used=0,
+        input_data=b'',
+        nonce=0,
+    )
+    receipt = EthereumTxReceipt(
+        tx_hash=evmhash,
+        contract_address=None,
+        status=True,
+        type=0,
+        logs=[],
+    )
+    dbethtx = DBEthTx(database)
+    with database.user_write() as cursor:
+        dbethtx.add_ethereum_transactions(cursor, [transaction], relevant_address=None)
+        events = evm_transaction_decoder.decode_transaction(
+            write_cursor=cursor,
+            transaction=transaction,
+            tx_receipt=receipt,
+        )
+    assert events == [
+        HistoryBaseEntry(
+            event_identifier=evmhash,
```

Comment:
Why isthere no gas event generated here?

Reply:
Yes, should be here, fixed. For some reason I thought that for deposits to exchanges we don't count gas (but we should count it of course)